### PR TITLE
Cleaned up all references to REV-1512's value prop experiment

### PR DESCRIPTION
**Pull request by inventhouse**

### CODE REVIEW

**Review by jhan217 — APPROVED** [review]
Looks fine to me, but I'd defer to @dianekaplan and @muselesscreator in case either of them still need the code.

File: src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx [hunk 1]
```diff
@@ -91,22 +81,7 @@ function SequenceNavigation({
       </Button>
       {renderUnitButtons()}
       {renderNextButton()}
-      {/** [MM-P2P] Experiment (additional conditional) */}
-      {!mmp2p.state.isEnabled && REV1512FlyoverEnabled
```
Comment (dianekaplan) [inline src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx:95]: I believe the new MM-P2P bit is from @muselesscreator, so let's check with him what he needs in this block. (Everything else is fine from my perspective)

Comment (inventhouse) [reply src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx:95]: I believe the "additional conditional" changes were meant to have the mmp2p flyover supersede the REV1512 one, and can be safely removed with REV experiment code.

Comment (muselesscreator) [reply src/courseware/course/sequence/sequence-navigation/SequenceNavigation.jsx:95]: ++

File: src/courseware/course/sequence/Sequence.jsx [hunk 2]
```diff
@@ -367,16 +195,10 @@ function Sequence({
...
-          {!mmp2p.state.isEnabled && REV1512FlyoverEnabled && isREV1512FlyoverVisible() && (
-            isMobile
-              ? <REV1512FlyoverMobile toggleREV1512Flyover={toggleREV1512Flyover} />
-              : <REV1512Flyover toggleREV1512Flyover={toggleREV1512Flyover} />
-          )}
+
           {/** [MM-P2P] Experiment */}
           {(mmp2p.state.isEnabled && mmp2p.flyover.isVisible) && (
-            isMobile
+            isMobile()
```
Comment (muselesscreator) [inline src/courseware/course/sequence/Sequence.jsx:201]: Nice, this is one of the places I was worried about.

**Review by muselesscreator — APPROVED** [review]

**Pull request merged**